src/search/search.component.html

```diff
@@ -10,6 +10,7 @@
 		'bx--toolbar-search--active': toolbar && active
 	}"
 	role="search"
+	aria-label="ariaLabel"
```

Comment:
This puts literal string `"ariaLabel"` for aria-label. It should probably be something like 
```suggestion
	[attr.aria-label]="ariaLabel"
```
to make it take the input value.

Reply:
oops my bad, muscle memory has me used to feeding an aria-label as an input to other components.